app/global-styles.scss

```diff
@@ -1,8 +1,11 @@
 @import 'custom-bootstrap-style';
 
-a {outline : none !important;}
+html { font-size: 15px; }
```

Comment:
based on default font size, I changed back to 15px. when Frans ready to start orvar project.. I'll change based on Frans opinion.